packages/components/icon/src/icon.tsx

```diff
@@ -21,22 +21,15 @@ export type IconProps = Omit<SVGAttributes<SVGElement>, keyof UIProps> &
  * @see Docs https://yamada-ui.com/components/media-and-icons/icon
  */
 export const Icon = forwardRef<IconProps, "svg">(
-  (
-    { as: element, viewBox, size: fontSize, className, __css, ...rest },
-    ref,
-  ) => {
+  ({ as: element, viewBox, size: fontSize, ...props }, ref) => {
+    const [styles, { className, ...rest }] = useComponentStyle("Icon", props)
```

Comment:
```
const [styles, { className, ...rest }] = useComponentStyle("Icon", props)
```

to

```
const [styles, mergedProps] = useComponentStyle("Icon", props)
const {
  className,
  as: element,
  viewBox,
  size: fontSize,
  ...rest
} = omitThemeProps(mergedProps)
```

Use `omitThemeProps` to exclude `props` from the theme.

Reply:
@hirotomoyamada 
```
型 'OmitProps<PropsOf<"svg">, "string" | "_hover" | "_nativeHover" | "_active" | "_nativeActive" | "_target" | "_notTarget" | "_nativeTarget" | "_focus" | "_nativeFocus" | ... 476 more ... | "size"> & ... 5 more ... & { ...; }' の引数を型 'ThemeProps' のパラメーターに割り当てることはできません。
  プロパティ 'size' の型に互換性がありません。
    型 'UIValue<number | (string & {}) | "small" | "-moz-initial" | "inherit" | "initial" | "revert" | "revert-layer" | "unset" | "large" | "medium" | "x-large" | "x-small" | "xx-large" | ... 17 more ... | "9xl"> | undefined' を型 'UIValue<string> | undefined' に割り当てることはできません。
      型 'number' を型 'UIValue<string> | undefined' に割り当てることはできません。
```

I get this error, can you tell me how to fix it?

Reply:
Add `ThemeProps<"Icon">`.

Reply:
Thank you very much!
Correction.